storage/database/dynamodb.go

```diff
@@ -114,7 +120,7 @@ func GetDefaultDynamoDBConfig() *DynamoDBConfig {
 	}
 }
 
-func NewDynamoDB(config *DynamoDBConfig) (*dynamoDB, error) {
+func NewDynamoDB(config *DynamoDBConfig, db string) (*dynamoDB, error) {
```

Comment:
How about changing `db` to `entryType` or `entryTypeString`?
Or you can pass `entryType DBEntryType` and then covert it inside of this function.

Reply:
review reflected